server/graphql/types/src/types/permissions.rs

```diff
@@ -33,6 +33,8 @@ pub enum UserPermission {
     OutboundShipmentMutate,
     InboundShipmentQuery,
     InboundShipmentMutate,
+    OutboundReturnQuery,
+    OutboundReturnMutate,
```

Comment:
As mentioned at stand up this morning - I think I've add the different permissions components in here, but there's something about needing to sync/fetch from central to update the user permissions, so they're not currently working...

We agreed to come back to permissions later, do them all in one go. Are we happy to leave these changes here or should I roll them back?

Reply:
This is fine, i've just added `Add permission checks, and sync them` [to epic](https://github.com/msupply-foundation/open-msupply/issues/2897)